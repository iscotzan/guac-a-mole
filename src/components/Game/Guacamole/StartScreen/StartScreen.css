.site-view {
  margin: 0; }
