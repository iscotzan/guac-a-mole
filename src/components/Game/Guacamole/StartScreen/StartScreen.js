// @flow
import * as React from "react";
import { withLocalize } from "react-localize-redux";
import { Translate } from "react-localize-redux";
import { Grid, Image } from "semantic-ui-react";
import { Link } from 'react-router-dom';

import "./StartScreen.css";
import "./drawBorderBtn.css";
const StartScreen = props => {
 
  return (
    <Grid columns={1} centered className={"site-view"}>
      <Grid.Row centered stretched>
        <Grid.Column verticalAlign="middle" stretched>
          <Image src="./../../assets/images/pexels-photo-1166419_med.jpg" fluid>
            <Link to="/guacamole"> 
            <button className={"btn draw-border shiny-btn"}>
              <Translate
                id="translations.startGameText"
              />
            </button>
            </Link>
          </Image>
        </Grid.Column>
      </Grid.Row>
    </Grid>
  );
};

export default withLocalize(StartScreen);
