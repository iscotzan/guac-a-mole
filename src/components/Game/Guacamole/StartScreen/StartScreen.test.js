import React from "react";
import StartScreen from "./StartScreen";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

const wrapper = shallow(
  <StartScreen />
);
describe("<StartScreen />", () => {
  it("should be defined", () => {
    expect(wrapper).toBeDefined();
  });

  it("renders without crashing", () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
