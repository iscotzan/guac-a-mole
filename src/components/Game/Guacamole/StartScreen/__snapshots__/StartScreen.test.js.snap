// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<StartScreen /> renders without crashing 1`] = `<[object Object] />`;
